### PR TITLE
Added Rotation around Z-axis to the defaults control.

### PR DESCRIPTION
**Pull request by qureshinomaan**
## Motivation and Context
Gives the agent 6 degrees of freedom.

### CODE REVIEW

File: habitat_sim/agent/controls/default_controls.py [hunk 1]
```diff
@@ -137,3 +137,19 @@ def __call__(self, scene_node: SceneNode, actuation_spec: ActuationSpec):
         _rotate_local(
             scene_node, -actuation_spec.amount, _X_AXIS, actuation_spec.constraint
         )
+        
+        
+@registry.register_move_fn(body_action=False)
+class LookClock(SceneNodeControl):
```
Comment (erikwijmans) [inline habitat_sim/agent/controls/default_controls.py:143]:
```suggestion
class RotateSensorClockwise(SceneNodeControl):
```

File: habitat_sim/agent/controls/default_controls.py [hunk 2]
```diff
@@ -137,3 +137,19 @@ def __call__(self, scene_node: SceneNode, actuation_spec: ActuationSpec):
...
+    def __call__(self, scene_node: SceneNode, actuation_spec: ActuationSpec):
+        _rotate_local(
+            scene_node, actuation_spec.amount, _Z_AXIS, actuation_spec.constraint
+        )
+
+
+@registry.register_move_fn(body_action=False)
+class LookAnti(SceneNodeControl):
```
Comment (erikwijmans) [inline habitat_sim/agent/controls/default_controls.py:151]:
```suggestion
class RotateSensorAntiClockwise(SceneNodeControl):
```

**Comment by Skylion007** [comment]
@qureshinomaan Follow the instructions for installing pre-commit so you don't have to worry about failing the linter after uploading. :)

**Comment by qureshinomaan** [comment]
> @qureshinomaan Follow the instructions for installing pre-commit so you don't have to worry about failing the linter after uploading. :)

Hey @Skylion007, Thanks! Will install before next commit for sure.

File: tests/test_controls.py [hunk 3]
```diff
@@ -173,6 +173,12 @@ def test_default_sensor_contorls(action, expected):
         look_down=habitat_sim.ActionSpec(
             "look_down", habitat_sim.ActuationSpec(amount=10.0)
         ),
+        rotate_sensor_clockwise=habitat_sim.ActionSpec(
```
Comment (erikwijmans) [inline tests/test_controls.py:176]: The action names and the expected result of executing the action also need to be added to the `default_sensor_control_testdata` list above for them to actually be tested.

**Comment by Skylion007** [comment]
@qureshinomaan Bump? It would be nice to have this merged in.

File: habitat_sim/agent/controls/default_controls.py (hunk 2 above)
Comment (Skylion007) [reply habitat_sim/agent/controls/default_controls.py]: Why is this AntiClockwise instead of CounterClockwise?

**Comment by Skylion007** [comment]
@qureshinomaan I went ahead and fixed it. You just had to run the `black` formatter on the repo.

**Pull request merged**